modules/dnn/src/layers/prior_box_layer.cpp

```diff
@@ -331,16 +335,26 @@ class PriorBoxLayerImpl CV_FINAL : public PriorBoxLayer
             Mat widths(1, _boxWidths.size(), CV_32FC1, &_boxWidths[0]);
             Mat heights(1, _boxHeights.size(), CV_32FC1, &_boxHeights[0]);
 
-            offsetsX.copyTo(umat_offsetsX);
-            offsetsY.copyTo(umat_offsetsY);
-            variance.copyTo(umat_variance);
-            widths.copyTo(umat_widths);
-            heights.copyTo(umat_heights);
+            if (use_half)
+            {
+                convertFp16(offsetsX, umat_offsetsX);
```

Comment:
All non-weights hyper-parameters should be in the origin precision.

Reply:
switched back to origin precision for non-weights parameter.